### PR TITLE
Setup continuous deployment

### PR DESCRIPTION
**Pull request by cbenz**
Connected to #38

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+deployment:
+  production:
+    branch: continuous-deployment
```
Comment (fpagnoux) [inline circle.yml]: Should not we deploy on `master` ?

Comment (cbenz) [inline circle.yml]: Good spot! I was testing and requested reviewing too fast.

Comment (fpagnoux) [inline circle.yml]: No worries, I know how annoying it is to test continuous deployment :)

**Comment by cbenz** [comment]
@fpagnoux I tested it with the `continuous-deployment` branch, but it should be the same with `master`.

CircleCI offers a very nice experience, with minimal configuration, supporting SSH keys.

**Pull request merged**